starters/tailwind/src/Disclosure.tsx

```diff
@@ -0,0 +1,115 @@
+  Heading,
+  Button,
+  DisclosureStateContext,
+} from "react-aria-components";
+import { tv } from "tailwind-variants";
+import { ChevronRight } from "lucide-react";
+import { composeTailwindRenderProps, focusRing } from "./utils";
+import { DisclosureGroupStateContext } from "react-aria-components";
+
+const disclosure = tv({
+  base: "group min-w-64 border dark:border-zinc-600 rounded-lg text-gray-900 dark:text-zinc-200",
+  variants: {
+    isInGroup: {
+      true: "border-0 border-b last:border-b-0 rounded-b-none last:rounded-b-lg",
+    }
+  }
+});
+
+const disclosureButton = tv({
+  extend: focusRing,
+  base: "rounded-lg flex gap-2 items-center w-full text-start p-2 cursor-default",
+  variants: {
+    isDisabled: {
+      true: 'text-gray-300 dark:text-zinc-600 forced-colors:text-[GrayText]'
+    },
+    isInGroup: {
+      true: "-outline-offset-2 rounded-none group-first:rounded-t-lg group-last:rounded-b-lg",
+    }
+  }
+});
+
+const chevron = tv({
+  base: "w-5 h-5 text-gray-500 dark:text-gray-400 transition-transform duration-200 ease-in-out",
+  variants: {
+    isExpanded: {
+      true: "transform rotate-90",
+    },
+    isDisabled: {
+      true: 'text-gray-300 dark:text-zinc-600 forced-colors:text-[GrayText]'
+    }
+  }
+});
+
+export interface DisclosureProps extends AriaDisclosureProps {
+  children: React.ReactNode;
+}
+
+export function Disclosure({ children, ...props }: DisclosureProps) {
+  let isInGroup = useContext(DisclosureGroupStateContext) !== null;
+  return (
+    <AriaDisclosure
+      {...props}
+      className={composeRenderProps(props.className, (className, renderProps) => disclosure({ ...renderProps, isInGroup, className }))}
+    >
+      {children}
+    </AriaDisclosure>
+  );
+}
+
+export interface DisclosureHeaderProps {
+  children: React.ReactNode;
+}
+
+export function DisclosureHeader({ children }: DisclosureHeaderProps) {
+  let { isExpanded } = useContext(DisclosureStateContext)!;
+  let isInGroup = useContext(DisclosureGroupStateContext) !== null;
+  return (
+    <Heading className="text-lg font-semibold">
+      <Button
+        slot="trigger"
+        className={(renderProps) => disclosureButton({ ...renderProps, isInGroup })}
+      >
+        {({isDisabled}) => (
+          <>
+            <ChevronRight aria-hidden className={chevron({ isExpanded, isDisabled })} />
+            {children}
+          </>
+        )}
+      </Button>
+    </Heading>
+  );
+}
+
+export interface DisclosurePanelProps extends AriaDisclosurePanelProps {
+  children: React.ReactNode;
+}
+
+export function DisclosurePanel({ children, ...props }: DisclosurePanelProps) {
+  return (
+    <AriaDisclosurePanel {...props} className={composeTailwindRenderProps(props.className, 'group-data-[expanded]:px-4 group-data-[expanded]:py-2')}>
+      {children}
+    </AriaDisclosurePanel>
+  );
+}
+
+export interface DisclosureGroupProps extends AriaDisclosureGroupProps {
+  children: React.ReactNode;
+}
```

Comment:
Should this include the `allowsMultipleExpanded` and `expandedKeys` option?

Reply:
They're on there, they just weren't showing in Controls because the DisclosureGroup story was in the Disclosure storybook file. I moved DisclosureGroup to it's own file so it's more obvious